### ISSUE TITLE
Problems with generation effects

### ISSUE THREAD

**Issue by leomeeeeesi** [VectorSpaceLab/OmniGen#123]
The generation effects in some usage scenarios are very poor. For example, dyeing someone's hair yellow, changing the number on a jersey, and so on. Can I solve these problems through training?

**Comment by staoxiao** [comment]
Yes, you can improve the performance for specific tasks by fine-tuning the model.
